tests/integration/test_models/test_full_battery_models/test_lithium_ion/test_compare_outputs_two_phase.py

```diff
@@ -155,11 +155,11 @@ def compare_outputs_two_phase_silicon_graphite(self, model_class):
             "Average negative secondary particle concentration",
         ]:
             np.testing.assert_allclose(
-                sol1[var].data[:20], sol2[var].data[:20], rtol=1e-2
+                sol[0][var].data[:20], sol[1][var].data[:20], rtol=1e-2
             )
 
         # More silicon means longer sim
-        self.assertLess(sol1["Time [s]"].data[-1], sol2["Time [s]"].data[-1])
+        self.assertLess(sol[0]["Time [s]"].data[-1], sol[1]["Time [s]"].data[-1])
```

Comment:
Time tests have been a bit problematic. I would recommend removing this part of the test.

Runtime can vary based on what else is running on the system, but the tests are run in parallel. So unless the time difference is quite large, the test will be fragile

Reply:
It could be replaced by testing "Discharge capacity [A.h]" instead to make it clearer